NetCore/NetCore.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="BenchmarkDotNet" Version="0.13.1.1695" />
     <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.1.1695" Condition="$([MSBuild]::IsOSPlatform('Windows'))" />
     <PackageReference Include="FreeImage.Standard" Version="4.3.8" />
+    <PackageReference Include="Imageflow.AllPlatforms" Version="0.9.0" />
```

Comment:
Should probably update all the other libraries to their latest and greatest too.

Reply:
Yeah that's sound advice :) I've just committed the change.